### DISCUSSION TITLE
Introducing ledokku CLI

### DISCUSSION THREAD

**Discussion by akhil-naidu** [ledokku/ledokku#398] [Ideas]
> ledokkuCLI is under the testing phase.

> You can use use the following commands in your `Local Linux PC` or `Cloud hosting VPS`

## How this script will Work
It Will be updated shortly

## Requirements for the ledokkuCLI
- Any distro which supports `bash`
- Pre-installed `wget`
- `root` access, when executing the script

## Getting Started
First, let's download the `ledokkuCLI` installer.
```
wget https://raw.githubusercontent.com/akhil-naidu/ledokku/new/ledokku-cli/ledokkucli.sh
```
Execute it with `root` user or `sudo` permissions; you will receive an error if you are not root.
```
sudo bash ledokkucli.sh
```
Lastly, use this command or close and reopen the terminal.
```
exec $SHELL
```
### A quick check
For now: type `ledo` and press the `tab key`; it should autofill to `ledokku` => this way you can confirm whether the ledokkuCLI was installed or not. 

## Usage of ledokkuCLI
For now, there is not much. Just use this command you will get an idea
```
ledokku help
```

**Comment by mulyadiyaminsiahaan** [comment]
Can you help me about how to instal and use ledokku?
